ui.frontend/src/view/index.js

```diff
@@ -20,8 +20,9 @@ import FormPanel from "./FormPanel.js";
 import FormTabs from "./FormTabs.js";
 import FormCheckBox from "./FormCheckBox.js";
 import FormFileInput from "./FormFileInput.js";
-import FormFileInputWidget from "./FormFileInputWidget.js";
+import FormFileInputWidgetBase from "./FormFileInputWidgetBase.js";
 import FormOptionFieldBase from "./FormOptionFieldBase";
+import FormFileInputWidget from "./FormFileInputWidget.js";
 
 
-export {FormField, FormContainer, FormFieldBase, FormPanel, FormTabs, FormCheckBox, FormOptionFieldBase, FormFileInput, FormFileInputWidget }
+export {FormField, FormContainer, FormFieldBase, FormPanel, FormTabs, FormCheckBox, FormOptionFieldBase, FormFileInput, FormFileInputWidgetBase, FormFileInputWidget }
```

Comment:
Why is base exposed when no one is using it ?

Reply:
Explained below!